### PR TITLE
restore logging for OS other than Windows

### PR DESCRIPTION
**Pull request by NLaws**
and update control parameters for latest FICO Xpress release 8.14.0

### CODE REVIEW

**Comment by NLaws** [comment]
addresses https://github.com/jump-dev/Xpress.jl/issues/166

**Comment by NLaws** [comment]
@joaquimg I guess that a maintainer will have to run the tests in order to get the secret for the Xpress license?

**Comment by NLaws** [comment]
Screen Shot 2022-05-20 at 10 14 58 AM
Screen Shot 2022-05-20 at 10 14 45 AM
Screen Shot 2022-05-20 at 10 14 33 AM

**Review by odow — APPROVED** [review]
Looks good, but I'll wait for @joaquimg to check.

File: src/MOI/MOI_wrapper.jl [hunk 1]
```diff
@@ -566,9 +566,8 @@ function reset_message_callback(model)
         removecbmessage(model.inner, C_NULL, C_NULL)
         model.message_callback = nothing
     end
-    if Sys.iswindows() &&                   # is win
-            model.inner.logfile == "" &&    # no file -> screen
-            model.log_level != 0            # has log
+    if model.inner.logfile == "" &&    # no file -> screen
```
Comment (joaquimg) [inline src/MOI/MOI_wrapper.jl:569]: Linux also requires this?

**Review by joaquimg — COMMENTED** [review]
Thanks!
Did you test on linux?

File: src/MOI/MOI_wrapper.jl (hunk 1 above)
Comment (NLaws) [reply src/MOI/MOI_wrapper.jl:569]: According to FICO:

> Prior to Xpress 8.9, the Optimizer library would automatically print messages to stdout for all Unix-like platforms
> (including MacOS) if no message callback function was defined. With Xpress 8.9 and later, the Optimizer library will no longer print messages automatically for any platform.
> 
> To print the messages produced by the Optimizer, a user should set a message callback, using XPRSaddcbmessage():
> 
> https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSaddcbmessage.html
> 
> and provide it with a function that prints each message to stdout, as in the example provided in the link above.

Comment (NLaws) [reply src/MOI/MOI_wrapper.jl:569]: What if you add:
```yaml
os: [ubuntu-latest, windows-latest, macOS-latest]
```
to the CI? Then all major platforms will be tested automatically.a

Comment (NLaws) [reply src/MOI/MOI_wrapper.jl:569]: > Linux also requires this?


Short answer: yes.

**Review by rafabench — APPROVED** [review]

**Comment by rafabench** [comment]
@joaquimg It's working on Xpress 8.11 on Linux too
image
image

**Pull request merged**